Makefile

```diff
@@ -36,7 +36,12 @@ debian/base/include/etc/confluent/docker/docker-utils.jar:
 	&& cd -
 
 build-debian: debian/base/include/etc/confluent/docker/docker-utils.jar
-	for component in ${COMPONENTS} ; do \
+	pushd debian \
+	&& rm -rf kafkacat \
+	&& git clone https://github.com/edenhill/kafkacat \
+	&& sed -ie "s/runtimeDeps='/runtimeDeps='openssl libsasl2-modules-gssapi-mit krb5-user krb5-config /" kafkacat/Dockerfile \
```

Comment:
I guess `libssl1.1` and `libsasl2-2` are omitted intentionally?

Reply:
Yeah -- it's not obvious, but the kafkacat Dockerfile already has them.